ansible_collections/arista/avd/molecule/cv_deploy/converge.yml

```diff
@@ -158,13 +157,13 @@
         cv_submit_workspace_force: true
         cv_run_change_control: true
 
-    - name: Test = Provision with cv_submit_workspace_force = false
+    - name: Provision with cv_submit_workspace_force = false
       run_once: true
       delegate_to: localhost
       ansible.builtin.import_role:
         name: arista.avd.cv_deploy
       vars:
-        cv_devices: [ci-s1-leaf1, ci-s1-leaf2, ci-s1-leaf3, ci-s1-leaf4, ci-s1-spine1, ci-s1-spine2, ci-s1-core1 ]
+        cv_devices: [ avd-ci-leaf2, avd-ci-core1 ]
```

Comment:
When trying to follow the guide to run the test I am not successful on the tenant getting issues (twice in a row) for task "Check CPV returns" line 176 below
![Screenshot 2024-09-23 at 16 31 34](https://github.com/user-attachments/assets/c1a82c60-83ee-42fb-b849-53de6ce136f6)

Can you please investigate? I have the workspaces on the tenant if you want to verify more

Also it is very hard to troubleshoot what step is failing as the names of the task when using `import_role` are hidden and so I am not sure where we are.

Suggestion would be to separate each "test" in a play in the same playbook or a whole different playbook.

Reply:
I guess running cv_deploy with `cv_submit_workspace_force = false` and expecting workspace submission to fail with `Failed to submit workspace` may be achieved by stopping streaming on one of the devices first. Otherwise workspace submission just succeeds without any errors